bot/src/discord/show_random_word.js

```diff
@@ -78,10 +78,13 @@ async function showRandomWord(
   }
   if (showStrokeOrder) {
     const kanji = word.match(/([\u4e00-\u9faf])/g);
-    await Promise.all(kanji.map(async (char) => { // shows the strokes of every kanji
-      const stroke = await createKanjiSearchPage(char, undefined, true);
-      await channel.createMessage(stroke);
+    const kanjiPages = await Promise.all(kanji.map(async (char) => {
+      createKanjiSearchPage(char, undefined, true);
     }));
```

Comment:
Hmm have you tested this? It doesn't look like it will work, because the kanji page isn't getting returned.

It could be:

```js
const kanjiPages = await Promise.all(kanji.map((char) => {
  return createKanjiSearchPage(char, undefined, true);
}));
```
Or
```js
const kanjiPages = await Promise.all(
  kanji.map((char) => createKanjiSearchPage(char, undefined, true)),
);
```

Reply:
Sorry, I had something similar to the second option but Lint didn't like it so I tweaked it a little bit and didn't test it afterwards.

With the first option I get:
```
Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`  arrow-body-style
```

And with the second
```
Unexpected newline after '('                                                               function-paren-newline
```

But if I leave it in one line, then it's too long.

I can do just this:
```
    const kanjiPages = await Promise.all(kanji.map(char =>
      createKanjiSearchPage(char, undefined, true)));
```

Reply:
Oh weird I thought ESLint would be okay with the second one. That's fine. Or you could make it two statements:

```js
const kanjiPagePromises = kanji.map((char) => createKanjiSearchPage(char, undefined, true));
const kanjiPages = await Promise.all(kanjiPagePromises);
```

Reply:
Is it possible that I have a different ESLint configuration? The first time I ran it I had to add this to .eslintrc.js so it would't throw thousands of line-break errors through the entire project:

```
      "linebreak-style": ["error", "windows"]
```

Maybe I have a too old (or too new) version of ESLint? I just checked and I have 6.14.8. I'm on Windows 10 by the way.

Reply:
Oh most likely you have git's autocrlf option enabled. On Windows that will convert line endings to Windows CR LF style on checkout. I like to disable that setting and use LF line endings even on Windows. I will add a `.gitattributes` file to automatically do that for Kotoba, should have done that a long time ago.

Reply:
Did that: https://github.com/mistval/kotoba/commit/bdfc18975b5eb52f834300e5caf79ae833b06c5e so if you clone again you shouldn't see those errors.